driver-core/src/main/com/mongodb/internal/connection/SocketChannelStream.java

```diff
@@ -50,11 +54,27 @@ public SocketChannelStream(final ServerAddress address, final SocketSettings set
         this.bufferProvider = notNull("bufferProvider", bufferProvider);
     }
 
+    private SocketChannel initializeSocketChannel() throws IOException {
```

Comment:
Nit: generally private method go after the method which invoke it.  So move it down below the open method.

Reply:
Done